File: 8_kyu/Gravity_Flip.rs
// Returns cubes sorted.
// if dir is 'R' sort ascending, else sort descending.
fn flip(dir: char, cubes: &[u32]) -> Vec<u32> {
    let mut result: Vec<u32> = cubes.to_vec();
    if dir == 'R' {
        result.sort_unstable()
    } else {
        result.sort_unstable_by(|a, b| b.cmp(a))
    }
    result
}

File: 8_kyu/Gravity_Flip.cs
using System;

public class Kata
{
    // Sorts array arr, ascending or descending if dir is 'L'.
    public static int[] Flip(char dir, int[] arr)
    {
        Array.Sort(arr);
        if( dir == 'L' ) Array.Reverse(arr);
        return arr;
    }
}